src/sdk/model/edge/certificates/x509-certificate.ts

```diff
@@ -0,0 +1,168 @@
+import { X509CertificateJson } from './__json__/x509-certificate-json';
+
+/* istanbul ignore next: autogenerated */
+export class X509Certificate {
+
+  constructor(private _json: X509CertificateJson) {
+  }
+
+  /**
+   * Get subject cn.
+   * @returns {string}
+   */
+  get subjectCn(): string {
+    return this._json.subject_cn;
+  }
+
+  /**
+   * Get issuer cn.
+   * @returns {string}
+   */
+  get issuerCn(): string {
+    return this._json.issuer_cn;
+  }
+
+  /**
+   * Get version.
+   * @returns {number}
+   */
+  get version(): number {
+    return this._json.version;
+  }
+
+  /**
+   * Get serial number.
+   * @returns {string}
+   */
+  get serialNumber(): string {
+    return this._json.serial_number;
+  }
+
+  /**
+   * Get signature algo name.
+   * @returns {string}
+   */
+  get signatureAlgoName(): string {
+    return this._json.signature_algo_name;
+  }
+
+  /**
+   * Get signature.
+   * @returns {string}
+   */
+  get signature(): string {
+    return this._json.signature;
+  }
+
+  /**
+   * Get not before.
+   * @returns {number}
+   */
+  get notBefore(): number {
+    return this._json.not_before;
+  }
+
+  /**
+   * Get not after.
+   * @returns {number}
+   */
+  get notAfter(): number {
+    return this._json.not_after;
```

Comment:
should be a date?

Reply:
yeah we can return a date instead of a timestamp. Good catch.